### PR TITLE
py sol for TripTastic

### PR DESCRIPTION
**Pull request by freakin23**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

AC: https://www.codechef.com/viewsolution/1110964545

### CODE REVIEW

File: solutions/silver/cc-TRPTSTIC.mdx [hunk 1]
```diff
@@ -119,4 +119,96 @@ int main() {
 ```
 
 </CPPSection>
+<PySection>
+
+```py
+def rangesum(row1: int, column1: int, row2: int, column2: int) -> int:
+	"""
+	:return the sum from (row1, column1) to (row2, column2)
+	Assumes row1 < row2 and column1 < column2
```
Comment (SansPapyrus683) [inline solutions/silver/cc-TRPTSTIC.mdx:128]: add an `assert` instead of just putting it in the docs

Comment (freakin23) [reply solutions/silver/cc-TRPTSTIC.mdx]: done.

**Comment by freakin23** [comment]
upd AC: https://www.codechef.com/viewsolution/1110983371

**Review by SansPapyrus683 — APPROVED** [review]
send

**Pull request merged**